biogeochem/EDCanopyStructureMod.F90

```diff
@@ -1389,9 +1389,9 @@ subroutine canopy_summarization( nsites, sites, bc_in )
                    call endrun(msg=errMsg(sourcefile, __LINE__))
                 end if
 
-                if(currentPatch%total_canopy_area-currentPatch%area.gt.1.0e-16)then
-                   write(fates_log(),*) 'too much canopy in summary',s, &
-                        currentPatch%nocomp_pft_label, currentPatch%total_canopy_area-currentPatch%area
+                if (currentPatch%total_canopy_area - currentPatch%area > area_error) then
```

Comment:
This increases the tolerance for this check.  I feel like keeping the check tight here is warranted given that it's coming at the end of fates dynamic run.  Maybe we should add another named tolerance?

Reply:
Sure I can put this back. In discussing this error with @rosiealice, @JessicaNeedham, and @ckoven they had said to relax this tolerance to 1E-9 but I can put this all back to what it was previously so we know this fix will be B4B